crates/bevy_time/src/time.rs

```diff
@@ -281,14 +281,14 @@ impl<T: Default> Time<T> {
     /// seconds.
     #[inline]
     pub fn delta_seconds(&self) -> f32 {
```

Comment:
We should rename these too actually.

Reply:
This probably wont be too fun to review :D